packages/server/src/comment/comment.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+import { CommentService } from './comment.service';
+import { Comment } from './comment.model';
+import { CreateCommentInput } from './comment.dto';
+
+@Resolver()
+export class CommentResolver {
+  constructor(private commentService: CommentService) {}
+
+  @Mutation(() => Comment)
+  async addComment(@Args('input') input: CreateCommentInput) {
+    return this.commentService.create(input);
+  }
+
+  @Mutation(() => Comment)
+  async deleteComment(@Args('id') id: string) {
+    return this.commentService.removeComment(id);
+  }
```

Comment:
Need to also have a way to resolve the list of replies since they are stored as a list of ids, but accessed as a list of comments